web/lib/utils/helpers.tsx

```diff
@@ -116,9 +118,29 @@ export const isStepCompleted = (step: Step, userData: UserData | undefined): boo
   });
 };
 
+export interface AlertProps {
+  variant: "success" | "warning" | "error";
+  body: string;
+  header?: string;
+  link?: string;
+}
```

Comment:
tbh, I feel weird that a "props" type is exported at all, but I see the reason. It still makes me think the type should like in the Alert file though? What do you think?